core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -460,4 +460,28 @@
    */
   boolean allowQualifyingCommonColumn();
 
+  /**
+   * Whether to allow the rewrite of the insert call in merge call.
+   *
+   * <p>For example, in the query
+   *
+   * <blockquote><pre>
+   * MERGE INTO t1 AS a
+   * USING t2 AS b
+   * ON a.x = b.x
+   * WHEN MATCHED THEN UPDATE SET
+   *   y = b.y
+   * WHEN NOT MATCHED THEN INSERT (x, y) VALUES (b.x, b.y)
+   * </pre></blockquote>
+   *
+   * <p>{@code VALUES (b.x, b.y)} is the inserted value. The default behavior
+   * is to rewrite it to {@code SELECT b.x, b.y FROM t2 AS b}. Which is not
+   * a valid grammar in some dialects.
+   *
+   * <p>Among the built-in conformance levels, false in
+   * {@link SqlConformanceEnum#BABEL},
+   * {@link SqlConformanceEnum#BIG_QUERY},
+   * true otherwise.
```

Comment:
nit: missing \</p> closing block corresponding to \<p> on line 481. Same for \<p> tags on lines 477 and 466.

Reply:
Good catch! Thanks for pointing it out. Fix in https://github.com/googleinterns/calcite/pull/139/commits/825b67fa1fb053d5c6c0edaffd9d20f3c284b553